standard/ERCs/erc-6900.md

```diff
@@ -137,9 +150,8 @@ interface IModularAccount {
 
     /// @notice Install a module to the modular account.
     /// @param module The module to install.
-    /// @param manifest the manifest describing functions to install
-    /// @param moduleInstallData Optional data to be decoded and used by the module to setup initial module data
-    /// for the modular account.
+    /// @param manifest the manifest describing functions to install.
+    /// @param moduleInstallData Optional data to be arbitrarily used by the account to handle the initial execution setup.
```

Comment:
What if we phrase it as "Optional data to be used by the account to handle the initial execution setup. Data encoding is implementation-specific." ? I think "arbitrarily use" has really a connotation of being really lax, and we still want to encourage similar-ish use.

Reply:
Makes sense! Addressing.

Reply:
Done in [8f719fd](https://github.com/erc6900/reference-implementation/pull/170/commits/8f719fdf3b73eb38a2c55252250d62d89cc806db)